sui/Docker.md

```diff
@@ -1,8 +1,9 @@
 # first build the image
-(cd ..; DOCKER_BUILDKIT=1 docker build --progress plain -f sui/Dockerfile.base -t sui .)
+#(cd ..; DOCKER_BUILDKIT=1 docker build --no-cache --progress plain -f sui/Dockerfile.base -t sui .)
```

Comment:
why not use the cache?

Reply:
When they do a new release,  I flush the cache while building a docker base image so really this line is used for cut/paste